src/ImageRunModal.jsx

```diff
@@ -332,6 +337,27 @@ export class ImageRunModal extends React.Component {
                     </Select.Select>
                 </div>
 
+                <label className='control-label' htmlFor='run-image-cpu-priority'>
+                    {_("CPU Priority")}
```

Comment:
I'm in favor of calling this "CPU Shares", to match the podman CLI and manpage. It also better points out that these are a relative number, not an absolute priority.